web/templates/settings/organization_settings_admin.hbs

```diff
@@ -114,6 +114,30 @@
                             <option value='{{this.id}}'>{{this.name}}</option>
                         {{/each}}
                     </select>
+
+                    <div class="dependent-settings-block" id="realm_jitsi_server_url_setting">
+                        <div>
+                            <label for="id_realm_jitsi_server_url" class="dropdown-title">
+                                {{t "Jitsi server URL" }}:
+                            </label>
+                            <select name="realm_jitsi_server_url" id="id_realm_jitsi_server_url" class="setting-widget prop-element settings_select bootstrap-focus-style" data-setting-widget-type="jitsi-server-url-setting">
+                                {{#if jitsi_server_url}}
+                                    <option value="server_default">{{ jitsi_server_url }}</option>
```

Comment:
Can we have this say `https://foo.bar (default)`, to make it clear this is the special default value.